src/main/java/gmailgdogra/service/ShiftReportGeneratingService.java

```diff
@@ -0,0 +1,31 @@
+package gmailgdogra.service;
+
+import gmailgdogra.pojo.OutputRow;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class ShiftReportGeneratingService {
+
+    public static XSSFWorkbook write(List<OutputRow> inSwipesCurrentShift) throws IOException {
+        FileInputStream templateFile = new FileInputStream(new File("src/main/resources/Shift Report Template.xlsx"));
```

Comment:
A good idea is to have such values as this filepath in a separate file, maybe even in a configuration file which you can easily edit without recompiling the application. It's not necessary, but think about it.

Reply:
Will have to find out how to best achieve it